Adding HTML tags especially <ul>, <li> to format issue description trough tasks

loganSIO opened the pull request.

loganSIO committed:
Adding HTML tags especially <ul>, <li> to format issue description trough tasks

loganSIO commented:
Hello @fcayre , je ne comprends pas pourquoi les tests ici ne passent pas. Ils passent en local chez moi.
Cela semble planter sur d'autres modules ?

fcayre commented:
@loganSIO je viens de relancer en mode debug, c'est bizarre ce qui se passe, c'est lié à https://github.com/fcayre/account-invoicing/commit/801e1fe45ca3458e552f7cfdcd35ec5a6a307097 mais on dirait que github actions n'a pas tiré ce patch en installant l'environnement de test.
Je te tiens au jus.

loganSIO commented:
> 

ça marche merci !!

fcayre commented:
@loganSIO compris désolé j'avais mal lu le log, c'était pourtant clair :-/ Les modifs que j'ai faites dans account-invoicing mentionnées ci-dessus impliquent des modifs dans commown-odoo-addons que je n'ai que dans ma branche 12.0-add-payment_token_uniquify. Je vais les passer en prod assez rapidement, du coup tu pourras rebaser tes devs dessus (sur la 12.0) et les tests se remettront à passer.

fcayre commented:
@loganSIO peux-tu tirer la 12.0, rebaser ta PR dessus et la repousser afin de relancer les tests STP ?

fcayre commented:
@loganSIO ah attends je vais essayer de le faire moi-même, je suis censé être autorisé ici et j'ai encore jamais essayé.

loganSIO commented:
> @loganSIO ah attends je vais essayer de le faire moi-même, je suis censé être autorisé ici et j'ai encore jamais essayé.

besoin d'encore une action @fcayre ?

loganSIO committed:
Adding the rest of issue_template as html

loganSIO commented:
@fcayre toutes les issues_templates sont faites ici.

fcayre commented on commown_self_troubleshooting/data/FP/screen/issue_template.xml:
```diff
@@ -6,34 +6,41 @@
+        <ul>
+          <li><t t-esc="model2"/></li>
+          <li><t t-esc="name"/></li>
+          <li><t t-esc="u'\n'.join((street, street2)).strip()"/></li>
```
u'\n'.join(...).strip() => "<br>".join(...)
sinon tout apparaîtra sur la même ligne ici.

loganSIO replied:
Fix dans ce commit : a4941065eaac4fd2c7114c2645084b8d218eb95f @fcayre

fcayre commented on commown_self_troubleshooting/data/FP/screen/issue_template.xml:
```diff
@@ -6,34 +6,41 @@
+            <p>après avoir effectué un auto-dépannage, il ressort qu'un écran muni d'une protection doit m'être envoyé.</p>
+          </t>
+          <t t-else="">
+            <p>après avoir effectué un auto-dépannage, il ressort qu'un nouvel appareil () doit m'être envoyé.</p>
```
On peut virer les parenthèses ici.

loganSIO replied:
Fix dans ce commit : a4941065eaac4fd2c7114c2645084b8d218eb95f @fcayre

fcayre commented on commown_self_troubleshooting/data/FP2/battery/issue_template.xml:
```diff
@@ -1,37 +1,56 @@
+                  <t t-esc="name"/>
+                </li>
+                <li>
+                  <t t-esc="u'\n'.join((street, street2)).strip()"/>
```
Idem

loganSIO replied:
Idem, fix dans ce commit : a4941065eaac4fd2c7114c2645084b8d218eb95f

fcayre commented on commown_self_troubleshooting/data/FP/screen/issue_template.xml:
```diff
@@ -6,34 +6,41 @@
+      </t>
+      <t t-if="more_info">
+        <p>J'ai porté à votre connaissance les informations complémentaires suivantes:</p>
+        <p><t t-esc="more_info"/></p>
```
J'ai un gros doute ici : more_info doit être du texte, et donc s'il contient des retours à la ligne ils ne seront pas pris en compte ici. Si c'est bien le cas il faudrait un `<pre>` et non un `<p>` autour.

À faire pour tous les more_info (après vérification hein !)

loganSIO replied:
Fix dans ce commit : a4941065eaac4fd2c7114c2645084b8d218eb95f @fcayre

fcayre commented on commown_self_troubleshooting/data/FP2/camera/issue_template.xml:
```diff
@@ -1,37 +1,62 @@
+                  <t t-esc="name"/>
+                </li>
+                <li>
+                  <t t-esc="u'\n'.join((street, street2)).strip()"/>
```
idem... (à vérifier partout)

loganSIO replied:
Idem : fix dans ce commit > a4941065eaac4fd2c7114c2645084b8d218eb95f

loganSIO committed:
Small improve on issues template

- Removing a () in data/FP/screen/issue_template.xml
- Extracted both street and split it into two lines
- Formatting more_info text box with <pre> to remove all the text on the same line

fcayre commented on commown_self_troubleshooting/data/Assistance/Generic Issue/issue_template.xml:
```diff
@@ -23,19 +23,29 @@
+            <ul>
+              <li t-esc="name"/>
+              <li t-esc="street"/>
+              <li t-esc="street2"/>
```
street2 is pretty often empty (and maybe other fields could?) so I would rather write:
`<li t-if="street2.strip()" t-esc="street2.strip()"/>`

loganSIO replied:
@fcayre c'est amendé toujours dans le commit d986895a12d7feb8c69c3ed2d7f1b3963a4c2a58

fcayre reviewed (changes requested):
We're almost there, would be cool to deploy this at the same time we deploy the B2B self troubleshooting matter.

fcayre commented:
/ocabot merge patch
